types/index.d.ts

```diff
@@ -39,6 +39,9 @@ declare class HCaptcha extends React.Component<HCaptchaProps, HCaptchaState> {
   resetCaptcha(): void;
   renderCaptcha(): void;
   removeCaptcha(): void;
+  setData(): void;
```

Comment:
The `setData` shall take a `data` param of type `Object`.

Reply:
The type shall be: @adekay 
```ts
setData(data: object): void;
```

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/0107edbc01dc916d77583cbed7f133a44689cc56